packages/design-system/src/components/Pagination/Page.test.tsx

```diff
@@ -0,0 +1,17 @@
+import Page from './Page';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+describe('Page', () => {
+    const onPageChange = jest.fn();
+
+    it('should render interactive el if not current', () => {
```

Comment:
I like the use of interactive vs. static in describing these tests. Makes it way less brittle!